src/common/Screen.js

```diff
@@ -64,7 +64,12 @@ class Screen extends PureComponent<Props> {
     return (
       <View style={[styles.screen, { marginBottom: safeAreaInsets.bottom }]}>
         <ZulipStatusBar />
-        <ModalBar title={title} searchBarOnChange={searchBarOnChange} />
+        { title ?
```

Comment:
This will break all screens with search functionality.
Your assumption that `title` missing its value is not a valid case is wrong.

Reply:
Apologies for the mistake, I have changed the commit. I added a new prop `hideTitle` to `Screen` that would hide the title if set to true, and used that instead.